controls/scheduler/built-in-dialogs.md

```diff
@@ -0,0 +1,96 @@
+
+![Telerik .NET MAUI Scheduler EditRecurrenceChoiceDialog](images/scheduler-dialogs-recurrencechoice.png)
+
+* **DeleteRecurrenceChoiceDialog**&mdash;Provides the end user the option to delete single occurrence of an appointment or the whole series.
+
+![Telerik .NET MAUI Scheduler DeleteRecurrenceChoiceDialog](images/scheduler-dialogs-deleterecurrencechoice.png)
+
+* **DeleteAppointmentChoiceDialog**&mdash;Available only on desktop. Provides a confirmation for deleting an appointment.
+
+![Telerik .NET MAUI Scheduler DeleteAppointmentChoiceDialog](images/scheduler-dialogs-deleteappointmentchoice.png)
+
+### Methods
+
+The Scheduler exposes a way to explicitly show the built-in dialogs through the following methods:
+
+* `CreateAppointmentWithDialog(DataRange range)`&mdash;Opens a dialog for creating an appointment.
+* `EditAppointmentWithDialog(Occurrence occurrence)`&mdash;Opens a dialog for editing an exsisting appointment.
+* `DeleteAppointmentWithDialog(Occurrence occurrence)`&mdash;Opens a dialog for deleting an existing appointment.
+
+Here is a quick example on how the methods for managing dialogs can be used:
+
+**1.** Create a sample view with a `RadScheduler` instance and a few buttons:
+
+<snippet id='scheduler-dialogs-methods-xaml' />
+
+**2.** Add the buttons' clicked event handlers in the code-behind:
+
+<snippet id='scheduler-dialogs-methods-eventhandlers' />
+
+### Events
+
+The Scheduler exposes the following events related to the built-in dialogs for managing appointments:
+
+* `DialogOpening`&mdash;occurs when a dialog is about to be opened. The `DialogOpening` event handler receives two parameters:
+    * The sender argument, which is of type `object`, but can be cast to the `RadScheduler` type.
+    * A `SchedulerDialogOpeningEventArgs` object which you can use to get the dialog type through `DialogType` property as well as cancel the opening by setting its `Cancel` property to `True`.
+* `DialogClosing`&mdash;occurs when a dialog is about to be closed. The `DialogClosing` event handler receives two parameters:
+    * The sender argument, which is of type `object`, but can be cast to the `RadScheduler` type.
+    * A `SchedulerDialogClosingEventArgs` object which you can use to get the dialog type through `DialogType` property as well as cancel the opening by setting its `Cancel` property to `True`. In addition, the `SchedulerDialogClosingEventArgs` provides a `DialogResult`(`bool?`) property which gives information on how the dialog is closed: `DialogResult` is null when the dialog is closed by the Close button of the dialog, `False` when Cancel option is pressed, otherwise the value is `True`.
+
+Check a quick example on how the `DialogOpening` and `DialogClosing` events can be used:
+
+**1.** Add the `RadScheduler` definition:
+
+<snippet id='scheduler-dialogs-events-xaml' />
+
+**2.** Add the event handlers:
+
+<snippet id='scheduler-dialogs-events' />
+
+### Customization
```

Comment:
Based on what users can achieve with the corresponding dialog styles, consider what the best name for the sections is.
Go with Customization, if users can use these styles to change the labels or further configure the UI elements of the built-in dialog (say, place them vertically, instead of horizontally, etc.).
Go with Styling, if it's just about colors, borders, font types, etc.

**It might also make sense to extract this into the Styling section and provide some brief examples of actual styling.**